File: 07_palindrome/Palindrome.java
public class Palindrome {
    public static void main(String[] args){

        String str = "1001";
        String reverse = "";
        int length = str.length();
        for(int i=0;i<length;i++){
            reverse = str.charAt(i) + reverse;
        }

        if(str.equals(reverse)){
            System.out.print("Palindrome");
        }
        else{
            System.out.print("Not a palindrome");
        }
    }
    
}

File: 07_palindrome/palindrome.cpp
#include<iostream>
using namespace std;

int main(){
    string str = "refere";
    string reverse = "";
    for(int i=0;i<str.length();i++){
        reverse = str[i] + reverse;
    }

    if(str == reverse){
        cout<<"palindrome"<<endl;
    }
    else{
        cout<<"not a palindrome"<<endl;
    }

    return 0;
}